src/Sql/Sql/DistributedAvailabilityGroup/Cmdlet/NewAzureSqlManagedInstanceLink.cs

```diff
@@ -68,40 +63,70 @@ public class NewAzureSqlManagedInstanceLink : AzureSqlManagedInstanceLinkCmdletB
-        [Parameter(Mandatory = true, ParameterSetName = CreateByParentObjectParameterSet, HelpMessage = "Name of the primary availability group.")]
+        [Parameter(Mandatory = true, ParameterSetName = CreateByNameParameterSet, HelpMessage = "Name of the partner availability group.")]
+        [Parameter(Mandatory = true, ParameterSetName = CreateByParentObjectParameterSet, HelpMessage = "Name of the partner availability group.")]
         [ValidateNotNullOrEmpty]
-        public string PrimaryAvailabilityGroupName { get; set; }
+        public string PartnerAvailabilityGroupName { get; set; }
 
         /// <summary>
-        /// Gets or sets the secondary availability group name
+        /// Gets or sets the managed instance availability group name
         /// </summary>
-        [CmdletParameterBreakingChangeWithVersion("SecondaryAvailabilityGroupName", "13.0.0", "6.0.0", ReplaceMentCmdletParameterName = "InstanceAvailabilityGroupName")]
-        [Parameter(Mandatory = true, ParameterSetName = CreateByNameParameterSet, HelpMessage = "Name of the secondary availability group.")]
-        [Parameter(Mandatory = true, ParameterSetName = CreateByParentObjectParameterSet, HelpMessage = "Name of the secondary availability group.")]
+        [Parameter(Mandatory = true, ParameterSetName = CreateByNameParameterSet, HelpMessage = "Name of the managed instance availability group.")]
+        [Parameter(Mandatory = true, ParameterSetName = CreateByParentObjectParameterSet, HelpMessage = "Name of the managed instance availability group.")]
         [ValidateNotNullOrEmpty]
-        public string SecondaryAvailabilityGroupName { get; set; }
+        public string InstanceAvailabilityGroupName { get; set; }
 
         /// <summary>
-        /// Gets or sets the target database
+        /// Gets or sets the target database/databases
         /// </summary>
-        [CmdletParameterBreakingChangeWithVersion("TargetDatabase", "13.0.0", "6.0.0", ChangeDescription = "The parameter 'TargetDatabase' is being replaced by parameter 'Databases'. The type of new parameter is changing from 'String' to 'List<String>'")]
-        [Parameter(Mandatory = true, ParameterSetName = CreateByNameParameterSet, HelpMessage = "Name of the target database.")]
-        [Parameter(Mandatory = true, ParameterSetName = CreateByParentObjectParameterSet, HelpMessage = "Name of the target database.")]
+        [Parameter(Mandatory = true, ParameterSetName = CreateByNameParameterSet, HelpMessage = "Database names in the distributed availability group.")]
+        [Parameter(Mandatory = true, ParameterSetName = CreateByParentObjectParameterSet, HelpMessage = "Database names in the distributed availability group.")]
         [ValidateNotNullOrEmpty]
-        public string TargetDatabase { get; set; }
+        public List<string> Databases { get; set; }
```

Comment:
The parameter here should be `Database` instead of `Databases` as singular form is also acceptable. The recommended type is `string[]`. Please change the code and update the docs accordingly.

Reply:
But what if user wants to add multiple databases, the Database would not make sense. This way we can say the user has databases list of 1 database.

Reply:
string[] can accept multiple databases as input as well. This is a common practice from PowerShell perspective. Please refer to https://raw.githubusercontent.com/Azure/azure-devtestlab/master/Samples/101-dtl-create-lab/azuredeploy.json

Reply:
Can't open the link. I was referring to name of the parameter not the type.

Reply:
I have discussed this with PM, can we leave it in plural ?

Reply:
No, you cannot leave it in plural unless the end users always need to pass in with multiple values. PowerShell users are accustomed to this convention, and deviating from it may cause confusion.

Reply:
Fixed.